dev.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+#1. Bundle
+
+pyinstaller main.py --clean --noconfirm \
```

Comment:
unnecessary line break?
i like the use of using full versions of arguments :100: